pandas/io/pytables.py

```diff
@@ -3635,23 +3645,21 @@ def _create_axes(
         data_columns=None,
         min_itemsize=None,
     ):
-        """ create and return the axes
-        legacy tables create an indexable column, indexable index,
-        non-indexable fields
-
-            Parameters
-            ----------
-            axes: a list of the axes in order to create (names or numbers of
-                the axes)
-            obj : the object to create axes on
-            validate: validate the obj against an existing object already
-                written
-            min_itemsize: a dict of the min size for a column in bytes
-            nan_rep : a values to use for string column nan_rep
-            encoding : the encoding for string values
-            data_columns : a list of columns that we want to create separate to
-                allow indexing (or True will force all columns)
+        """
+        Create and return the axes.
 
+        Parameters
+        ----------
+        axes: a list of the axes in order to create (names or numbers of the axes)
```

Comment:
can you edit these to be proper doc-strings

Reply:
formalized it to the extent I could.  A lot of these docstrings could use improvement